src/io/flutter/view/InspectorPanel.java

```diff
@@ -653,40 +658,43 @@ protected void customizeCellRenderer(JTable table, @Nullable Object value, boole
       final CompletableFuture<Map<String, InstanceRef>> propertiesFuture = node.getValueProperties();
       if (propertiesFuture.isDone() && !propertiesFuture.isCompletedExceptionally()) {
         final Map<String, InstanceRef> properties = propertiesFuture.getNow(null);
-        switch (node.getPropertyType()) {
-          case "Color": {
-            final int alpha = getIntProperty(properties, "alpha");
-            final int red = getIntProperty(properties, "red");
-            final int green = getIntProperty(properties, "green");
-            final int blue = getIntProperty(properties, "blue");
-
-            //noinspection UseJBColor
-            final Color color = new Color(red, green, blue, alpha);
-            this.setIcon(colorIconMaker.getCustomIcon(color));
-            if (alpha == 255) {
-              append(String.format("#%02x%02x%02x", red, green, blue), textAttributes);
-            }
-            else {
-              append(String.format("#%02x%02x%02x%02x", alpha, red, green, blue), textAttributes);
-            }
+        final String propertyType = node.getPropertyType();
+        if (propertyType != null) {
+          switch (propertyType) {
```

Comment:
this diff is just increasing indent by 2 and adding the propertyType != null check to avoid throwing exceptions for property objects that aren't really properties (e.g. new DiagnosticNode.message()